src/main/java/com/ns/task/common/Product.java

```diff
@@ -0,0 +1,40 @@
+package com.ns.task.common;
+
+public class Product {
```

Comment:
Would you find useful to add toString, hashCode and equals methods to this class?
If so, why?, and find the best practices for hashCode and Equals.